<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Secure USB backup</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayoutMainWindow">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidgetRoot">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_computer">
       <attribute name="title">
        <string>Computer</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayoutComputer">
        <item row="0" column="0">
         <widget class="QLabel" name="computerLabel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Computer structure</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="backupButtonComp">
          <property name="text">
           <string>Backup</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="helpButton">
          <property name="text">
           <string>Help</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacerComputer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>242</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="scanButton">
          <property name="text">
           <string>Scan</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="fileTypesButton">
          <property name="text">
           <string>File types</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="filesFoundLabel">
          <property name="text">
           <string>Files found</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="6">
         <widget class="QTreeView" name="treeViewComputer"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_storage">
       <attribute name="title">
        <string>HDD Storage</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayoutStorage">
        <item row="4" column="1">
         <spacer name="verticalSpacerStorage">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="storageStructLabel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Storage structure </string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="restoreButton">
          <property name="text">
           <string>Restore</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="backupButtonStorage">
          <property name="text">
           <string>Backup</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QProgressBar" name="progressBarBackup">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="helpButtonStorage">
          <property name="text">
           <string>Help</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="5">
         <widget class="QTreeView" name="treeViewStorage"/>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="newStorageButton">
          <property name="text">
           <string>New storage</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
